packages/mobile/src/backup/BackupIntroduction.tsx

```diff
@@ -97,16 +99,19 @@ class BackupIntroduction extends React.Component<Props> {
     return (
       <SafeAreaView style={styles.container}>
         <ScrollView contentContainerStyle={styles.scrollContainer}>
-          <Image source={backupIcon} style={styles.logo} />
-          <Text style={styles.h1}>{t('backupAndRecovery')}</Text>
           {!backupCompleted && (
             <>
+              <BackupIntroIcon style={styles.logo} width={180} height={138} />
```

Comment:
nit: what happens if you specify a width and height with a different ratio than the source svg?
wondering if we should compute the other dimension to prevent incorrect usages.

Reply:
Yeah I tested this. The SVG maintains it's scale but then includes a extra whitespace based on the values you set. Which may be helpful in certain cases so I think it'd be good to still let devs set the sizes